e2e/cypress/integration/keyboard_shortcuts/keyboard_shortcuts_two_spec.js

```diff
@@ -0,0 +1,269 @@
+
+        // * Verify that the recently posted message is shown in the RHS
+        cy.getLastPostId().then((postId) => {
+            cy.get(`#rhsPostMessageText_${postId}`).should('exist');
+        });
+    });
+
+    it('MM-T1279 - Keyboard shortcuts menu item', () => {
+        cy.get('#channel-header').should('be.visible').then(() => {
+            // # Click hamburger main menu
+            cy.get('#channelHeaderUserGuideButton').click();
+            cy.get('.dropdown-menu').should('be.visible').then(() => {
+                // # Select 'Keyboard Shortcuts'
+                cy.get('#keyboardShortcuts').should('be.visible');
+                cy.get('#keyboardShortcuts button').click();
+                cy.get('#shortcutsModalLabel').should('be.visible');
+            });
+        });
+    });
+
+    it('MM-T1575 - Ability to Switch Teams', () => {
+        const count = 10;
+        const teamNames = [];
+        const teamDisplayNames = [];
+        const channelNames = [];
+        const channelDisplayNames = [];
+
+        // Step #1
+        cy.apiLogout();
+        cy.apiLogin(testUser);
+        cy.visit(`/${testTeam.name}/channels/town-square`).wait(TIMEOUTS.HALF_SEC);
+
+        for (let index = 0; index < count; index++) {
+            teamNames.push('team' + index);
+            teamDisplayNames.push('Team' + index);
+            channelNames.push('channel' + index);
+            channelDisplayNames.push('Channel' + index);
+
+            cy.apiCreateTeam(teamNames[index], teamDisplayNames[index]).then(({team}) => {
+                teamNames[index] = team.name;
+                teamDisplayNames[index] = team.display_name;
+                cy.apiCreateChannel(team.id, channelNames[index], channelDisplayNames[index]).then(({channel}) => {
+                    channelNames[index] = channel.name;
+                    channelDisplayNames[index] = channel.display_name;
+                    cy.visit(`/${team.name}/channels/${channel.name}`).wait(TIMEOUTS.HALF_SEC);
+                });
+            });
+        }
+
+        for (let index = 0; index < count; index++) {
+            // # Verify that we've switched to the correct team
+            cy.get('#headerTeamName').should('contain', teamDisplayNames[count - index - 1]);
+
+            // # Verify that we've switched to the correct channel
+            cy.get('#channelHeaderTitle').should('be.visible').should('contain', channelDisplayNames[count - index - 1]);
+
+            // # Press CTRL/CMD+SHIFT+UP
+            if (cy.isMac()) {
+                cy.get('body').type('{cmd}{option}', {release: false}).type('{uparrow}').type('{cmd}{option}', {release: true});
+            } else {
+                cy.get('body').type('{ctrl}{shift}', {release: false}).type('{uparrow}').type('{ctrl}{shift}', {release: true});
+            }
+        }
+
+        // Step #2 - not clear what are the "new shortcuts" - verifying that 'Keyboard Shortcuts' show
```

Comment:
I believe it's to verify if shortcuts for "Previous Team" "Next Team" and "Switch to a specific team" are displayed in Keyboard shortcuts dialog. I've updated the test case.
<img width="306" alt="Screen Shot 2020-12-02 at 11 51 25 PM" src="https://user-images.githubusercontent.com/5334504/100896823-011bce80-34fa-11eb-9ae1-5c6373341631.png">